README.md

```diff
@@ -84,7 +89,8 @@ At present, the [image classification](https://webmachinelearning.github.io/webn
 
 * **Notes**:
 
-  Running WebNN on NPU requires a higher version of DirectML.dll than the one in the Windows system. Using the `--use-redist-dml` flag will allow Google Chrome Canary to load the downloaded DirectML.dll with a sufficiently high version.
+  1. Running WebNN on NPU requires a higher version of DirectML.dll than the one in the Windows system. Using the `--use-redist-dml` flag will allow Google Chrome Canary to load the downloaded DirectML.dll with a sufficiently high version.
+  2. Since the NPU drivers may be blocked by default if they are not the latest [Intel® Core™ Ultra NPU Driver on Windows](https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html), you can use the `--disable_webnn_for_npu=0` flag to bypass. However, the functionality is not guaranteed.
```

Comment:
Do we really need `However, the functionality is not guaranteed.`? This is known to users by default before officially shipping WebNN.

Reply:
I just intend to be responsible for these new added notes, to notice that if you follow these notes, you may meet functionality issues.

Agree that the `However, the functionality is not guaranteed` is negative, I am OK to remove them if you think that's really not necessary.